### PR TITLE
D3ASIM-3066: change set_power_forecast to set_energy_forecast

### PR DESCRIPTION
**Pull request by Hassan754**
[Ticket](https://gridsingularity.atlassian.net/browse/D3ASIM-3066)
### Description
Changed `set_power_forecast` functions / endpoints to `set_energy_forecast`

## Refs
Related PRs :
[Api-client , MQTT ](https://github.com/gridsingularity/d3a-api-client/pull/77)
[d3a-web](https://github.com/gridsingularity/d3a-web/pull/1095)

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -73,6 +73,7 @@ deps =
 	coverage
 commands =
 	python -c "import fcntl; fcntl.fcntl(1, fcntl.F_SETFL, 0)"
+	pip install pip==20.3.1
```
Comment (hannesdiedrich) [inline tox.ini:76]: Why is this needed again? Probably a remnant from before, right?

Comment (Hassan754) [reply tox.ini:76]: Yes , I'll remove it from here and pin it in the requirements , this is the only uncovered one

**Review by faizan2590 — APPROVED** [review]

File: requirements/base.txt [hunk 2]
```diff
@@ -27,4 +27,4 @@ sortedcontainers==2.3.0   # via -r requirements/base.in
 unidecode==0.4.21         # via awesome-slugify
 
 # The following packages are considered to be unsafe in a requirements file:
-# pip
+pip==20.3.1               # via -r requirements/base.in, pip-tools
```
Comment (hannesdiedrich) [inline requirements/base.txt:30]: This is again a remnant of our old issue right?

Comment (Hassan754) [reply requirements/base.txt:30]: Yes right @hannesdiedrich

**Review by hannesdiedrich — COMMENTED** [review]
One comment, LGTM once resolved

File: requirements/base.in [hunk 3]
```diff
@@ -1,3 +1,4 @@
+pip==20.3.1
```
Comment (hannesdiedrich) [inline requirements/base.in:1]: Please also remove this. Thanks

Comment (hannesdiedrich) [reply requirements/base.in:1]: Sorry, just saw your other comment, Why is it needed to be pinned?

Comment (Hassan754) [reply requirements/base.in:1]: @hannesdiedrich when we compile the requirements, it is important to have it pinned in the .in files , this will be translated to the .txt files

Comment (hannesdiedrich) [reply requirements/base.in:1]: I see now, didnt remember that we pinned it to 20.3.1 everywhere I thought we always install the latest version now. OK then. leave it as is

**Review by hannesdiedrich — APPROVED** [review]
LGTM
